test/foundry/BullaFrendLend/BullaFrendLend.t.sol

```diff
@@ -660,4 +674,311 @@ contract TestBullaFrendLend is Test {
         assertEq(tokenURI, "ipfs://QmTestTokenURI", "Token URI not correctly stored on claim");
         assertEq(attachmentURI, "ipfs://QmTestAttachmentURI", "Attachment URI not correctly stored on claim");
     }
+    
+    function testSetProtocolFee() public {
+        uint256 newProtocolFeeBPS = 2000; // 20%
+        
+        // Test that non-admin cannot set protocol fee
+        vm.prank(debtor);
+        vm.expectRevert(abi.encodeWithSelector(NotAdmin.selector));
+        bullaFrendLend.setProtocolFee(newProtocolFeeBPS);
+        
+        vm.prank(admin);
+        bullaFrendLend.setProtocolFee(newProtocolFeeBPS);
+        
+        assertEq(bullaFrendLend.protocolFeeBPS(), newProtocolFeeBPS, "Protocol fee not updated correctly");
+        
+        // Verify fee calculation with new rate
+        uint256 amount = 1 ether;
+        uint256 expectedFee = (amount * newProtocolFeeBPS) / 10000; // 0.2 ether
+        assertEq(bullaFrendLend.calculateProtocolFee(amount), expectedFee, "Protocol fee calculation incorrect after update");
```

Comment:
this is just duplicating contract logic again, testing nothing.

The asserts above are sufficient